src/main/kotlin/com/sourcegraph/cody/agent/CodyAgent.kt

```diff
@@ -129,6 +129,7 @@ private constructor(
                               webview = ClientCapabilities.WebviewEnum.Native,
                               webviewNativeConfig =
                                   WebviewNativeConfigParams(
+                                      view = WebviewNativeConfigParams.ViewEnum.Multiple,
```

Comment:
speaking of the devil... our protocol was outdated. 

From now on, we will be able to notice similar scenarios in a PR bumping the cody commit.